aptos-move/framework/aptos-framework/sources/native_bridge.move

```diff
@@ -173,19 +200,20 @@ module aptos_framework::native_bridge {
 
         // Emit the event
         let bridge_events = borrow_global_mut<BridgeEvents>(@aptos_framework);
-        event::emit_event(  
+        event::emit_event(
             &mut bridge_events.bridge_transfer_completed_events,
-            BridgeTransferCompletedEvent {  
-                bridge_transfer_id,  
-                initiator,  
-                recipient,  
-                amount,  
-                nonce,  
-            },  
-        );  
-    } 
+            BridgeTransferCompletedEvent {
+                bridge_transfer_id,
+                initiator,
+                recipient,
+                amount,
+                nonce,
+            },
+        );
+    }
 
-    #[test(aptos_framework = @aptos_framework, sender = @0xdaff)]
+
+    #[test(aptos_framework = @aptos_framework, sender = @0x726563697069656e740000000000000000000000000000000000000000000000)]
```

Comment:
is this the hex equivalent of 0xcafe?

Reply:
That `0x726563697069656e74` address is the word "recipient" as hex. just an arbitrary word that i had used in the solidity tests so I wanted to keep it the same to produce the same bridge transfer IDs as in Solidity. 

I use it that same value as the initiator in the L2 -> L2 test... guess I could make it another word in case it's misleading, but I don't think anyone would know it means "recipient" unless we tell them. It's just meant to be some Move address.